### PR TITLE
Feature/issue 3268 test continuous changes subscription 

### LINKED ISSUE #3268: Integration test timeout in TestContinuousChangesSubscription
The integration tests timed out after 30 minutes when running `TestContinuousChangesSubscription`

http://uberjenkins.sc.couchbase.com/view/Build/job/sync-gateway-integration-master/366

### PR DESCRIPTION
**Pull request by tleyden**
Fixes #3268 

Companion PR for https://github.com/couchbase/go-blip/pull/20

- [x] Merge https://github.com/couchbase/go-blip/pull/20
- [x] Update manifest.xml
- [ ] Merge this PR

### CODE REVIEW

File: rest/blip_api_test.go [hunk 1]
```diff
@@ -219,7 +219,7 @@ func TestContinuousChangesSubscription(t *testing.T) {
 	subChangesResponse := subChangesRequest.Response()
 	assert.Equals(t, subChangesResponse.SerialNumber(), subChangesRequest.SerialNumber())
 
-	for i := 1; i < 1500; i++ {
+	for i := 1; i < 150000; i++ {
```
Comment (tleyden) [inline rest/blip_api_test.go]: This change caused the test to reliably repro the issue, and it still ran in  a few seconds for me locally.

**Comment by tleyden** [comment]
Integration test:

http://uberjenkins.sc.couchbase.com/view/Build/job/sync-gateway-integration-master/416/

**Comment by tleyden** [comment]
I need to look into why the drone ci build failed.

Kicked off an integration test with the race flag set: http://uberjenkins.sc.couchbase.com/view/Build/job/sync-gateway-integration-master/437/

**Comment by tleyden** [comment]
@adamcfraser this one should be ready to review -- currently the only change in this PR is to point the manifest to a later version of go-blip, to uptake https://github.com/couchbase/go-blip/pull/20

Initially I increased the number of revs pushed in the test to reliably repro, but I ended up rolling that change back because it made the test take much longer to run (I suspect that may have been why an earlier commit failed the drone ci check)

**Comment by tleyden** [comment]
> Initially I increased the number of revs pushed

I just spent some more time investigating this, and while bumping the revs can repro the issue sometimes, it's fairly sporadic and takes an extremely long time under `-race`.  (I'm not even sure how long, because I killed it after a few minutes)

Even without the `-race` flag, it takes 45s.  I don't think this repro case belongs in a unit test.. and should be moved to a different type of test like a functional test or a system test.  

Also, the underlying https://github.com/couchbase/go-blip/pull/20 issue added tests which reproduce the issue at a lower level.

**Comment by djpongh** [comment]
Already up to current version of go-blip

**Pull request closed without merging**